app/navigation/houseConsignment/index/items/HouseConsignmentItemNavigator.scala

```diff
@@ -59,6 +59,29 @@ class HouseConsignmentItemNavigator extends Navigator {
           controllers.houseConsignment.index.items.routes.AddCombinedNomenclatureCodeYesNoController
             .onPageLoad(ua.id, houseConsignmentIndex, itemIndex, NormalMode)
         )
+    case PackageTypePage(houseConsignmentIndex, itemIndex, packageIndex) =>
+      ua =>
+        Some(
+          controllers.houseConsignment.index.items.packages.routes.AddNumberOfPackagesYesNoController
+            .onPageLoad(ua.id, NormalMode, houseConsignmentIndex, itemIndex, packageIndex)
+        )
+
+    case NumberOfPackagesPage(houseConsignmentIndex, itemIndex, packageIndex) =>
+      ua =>
+        Some(
+          controllers.houseConsignment.index.items.packages.routes.AddPackageShippingMarkYesNoController
+            .onPageLoad(ua.id, houseConsignmentIndex, itemIndex, packageIndex, NormalMode)
+        )
+
+    case AddPackageShippingMarkYesNoPage(houseConsignmentIndex, itemIndex, packageIndex) =>
+      ua => addPackageShippingMarkYesNoRoute(ua, ua.id, houseConsignmentIndex, itemIndex, packageIndex, NormalMode)
+
+    case PackageShippingMarkPage(houseConsignmentIndex, itemIndex, _) =>
+      ua =>
+        Some(
+          controllers.houseConsignment.index.items.packages.routes.AddAnotherPackageController.onPageLoad(ua.id, houseConsignmentIndex, itemIndex, NormalMode)
+        )
```

Comment:
Again, any reason why this can't be in https://github.com/hmrc/manage-transit-movements-unloading-frontend/blob/main/app/navigation/houseConsignment/index/items/PackagesNavigator.scala?